devdoc/Metrics.rst

```diff
@@ -306,29 +316,92 @@ metrics
 metrics.sinks
     List of sinks defined for the component (may have been combined with global config)
 
-metrics.sinks.sink
-    The repeating element
-
 metrics.sinks.sink.type
```

Comment:
remove ".sink", possibly metrics.sinks[].type for clarity.  Same for following definitions.